packages/chaire-lib-backend/src/utils/processManagers/TrRoutingProcessManager.ts

```diff
@@ -67,7 +75,8 @@ const start = async (parameters: { port?: number; debug?: boolean; cacheDirector
     const port = parameters.port || Preferences.get('trRouting.port');
 
     // TODO Check why we need this await, should not be useful before returning
-    return await startTrRoutingProcess(port, undefined, parameters);
+    // Set 2 threads as default, just in case we have more than one request being handled
```

Comment:
This comment will be wrong if we change the default to 3... The comment should be where you actually set the DEFAULT_THREAD_COUNT above

Reply:
Yeah, forgot to move it after adding the constant